src/models/index.ts

```diff
@@ -0,0 +1,67 @@
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
```

Comment:
We recently upgraded the kubevirt-api, so you should be able to import all those models from `@kubevirt-ui/kubevirt-api/console/models/` directory